docs/product/explore/feature-flags/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Feature Flags"
+sidebar_order: 100
+description: "Learn how to set up and interact with Sentry's feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
+
+</Alert>
+
+Link your external feature flag integrations with Sentry to provide feature flag insights right inside the Sentry UI. Linking one or more integrations will allow you to view recent flag evaluations in one place and identify potential suspect flags related to errors. In addition, Sentry will provide insights on feature flag updates relative to error event timelines.
```

Comment:
Hmmm, I think we should speak to the value prop a little more on the product index page.

- list of feature flags and their 'evaluated' value at the time of an individual error event (in evaluation order) **to help developers reproduce the user's application state** 
feature flag changes (i.e., when a new flag is created or conditions of a flag change) to overlay on top of an error event volume chart **to understand if there is any correlation between feature flag changes and errors**